e2e/tests/transfer/base_test.go

```diff
@@ -29,34 +29,25 @@ type TransferTestSuite struct {
 	testsuite.E2ETestSuite
 }
 
+// transferSelfParamsFeatureReleases represents the releases the transfer module started managing its own params.
+var transferSelfParamsFeatureReleases = semverutil.FeatureReleases{
+	MajorVersion: "v8",
+}
```

Comment:
feature release matrices are being grouped together in test values in #3558, just noting this for resolving merge conflicts